app/controllers/package_set.js

```diff
@@ -49,6 +49,15 @@ export default Controller.extend({
       return record.get("isSet") ? record.get("packages") : [record];
     }
   ),
+  packageUnavailableInSet: computed(
+    "allPackages.@each.availableQuantity",
+    function() {
+      let quantity = this.get("allPackages").any(
+        pkg => pkg.get("availableQuantity") == 0
+      );
+      return !!quantity;
+    }
+  ),
 
   notAvailableInStock: observer(
```

Comment:
Since you are now computing `packageUnavailableInSet`, you should probably remove `notAvailableInStock` since it's manually setting that property in an observer.

I'm not sure why it was an observer to begin with, computed property is definitely the way to go 👍

Reply:
sure..done